src/main/java/pl/futurecollars/invoicing/db/file/IdService.java

```diff
@@ -6,11 +6,13 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import org.springframework.stereotype.Repository;
 import pl.futurecollars.invoicing.utils.FilesService;
 
 @NoArgsConstructor
 @AllArgsConstructor
 @Data
+@Repository
```

Comment:
Tutaj ta adnotacja nie jest już potrzebna, bo masz @Bean na tym serwisie w DatabaseConfiguration - który też inicjuje komponent springowy. Teraz pod spodem tworzysz dwa beany, a potem Spring nadpisuje jednego drugim. Jak włączysz sobie logowanie debug w springu, to w logach zobaczysz coś takiego:
 `Overriding bean definition for bean 'idService' with a different definition: replacing...`